src/index.ts

```diff
@@ -0,0 +1,71 @@
+import 'dotenv/config'
+import express, { Express, Request, Response } from 'express';
+import { AssetsRequestBody } from './models/AssetType';
+import { CacheDataAssetKey } from './models/CacheType';
+import { supportedNetworks, SupportedNetworks } from './models/Networktype';
+import TokenRegistryService from './services/TokenRegistryService';
+
```

Comment:
add some code here to load a config file which contains all the different config parameters